Source/Core/VideoBackends/D3D/D3DState.cpp

```diff
@@ -242,6 +243,25 @@ void StateManager::SetComputeShader(ID3D11ComputeShader* shader)
 
 void StateManager::SyncComputeBindings()
 {
+  if (m_last_compute_rtv_count != m_pending.framebuffer->GetNumRTVs())
+  {
+    std::vector<ID3D11RenderTargetView*> rtv_null(m_pending.framebuffer->GetNumRTVs());
+    if (g_ActiveConfig.backend_info.bSupportsBBox)
+    {
+      ID3D11UnorderedAccessView* uav = nullptr;
+      D3D::context->OMSetRenderTargetsAndUnorderedAccessViews(
+          m_pending.framebuffer->GetNumRTVs(), rtv_null.data(), nullptr,
+          m_pending.framebuffer->GetNumRTVs() + 1, 1, &uav, nullptr);
+    }
+    else
+    {
+      D3D::context->OMSetRenderTargets(m_pending.framebuffer->GetNumRTVs(), rtv_null.data(),
+                                       nullptr);
+    }
+    m_dirtyFlags |= DirtyFlag_Framebuffer;
+    m_last_compute_rtv_count = m_pending.framebuffer->GetNumRTVs();
+  }
+
```

Comment:
Is there a reason you need to unbind everything to run a compute shader?

Reply:
Now that you mention it, this is unnecessary.  Not sure why I initially had it.  I did see an error in validation errors:

> The Shader Resource View dimension declared in the shader code (TEXTURE2DARRAY) does not match the view type bound to slot 0 of the Pixel Shader unit (BUFFER)

I notice that our lovely bounding box has type buffer.  I'm not sure why my changes would impact error that though.  I need to test master.  Regardless, seems there's a flaw somewhere.  I will investigate.

Reply:
@TellowKrinkle - thanks for looking this over.  I took this out, the same validation error happens on master with gpu texture decoding on.  We should resolve it but it at least doesn't impact this review!